### ISSUE TITLE
[a11y] missing header for actions column

### ISSUE THREAD

**Issue by susnux** [nextcloud/groupfolders#4464]
### How to use GitHub

* Please use the 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to show that you are affected by the same issue.
* Please don't comment if you have no relevant information to add. It's just extra noise for everyone subscribed to this issue.
* Subscribe to receive notifications on status change and new comments. 

### Steps to reproduce
1. open admin settings
2. inspect table to see header of table (last column)

### Expected behaviour
Have proper heading (even if `hidden-visually`).

### Actual behaviour
No header at all, bad for screen reader navigation.

<img width="587" height="489" alt="Image" src="https://github.com/user-attachments/assets/b721172d-fcf4-4d57-873b-a00405165a22" />
